### PR TITLE
Craftable muskets + Flintlocks

### PR DESCRIPTION
**Pull request by Tryded**
## About the PR

Makes a Musket meant for DeltaV with its own ammo and crafting recipe.

The Musket has multiple parts that all have to be individually crafted to make the musket.
## Why / Balance

Line Infantry will be funny and using the weapon by yourself is kind of bad as it takes 8 seconds to fire again. You can dual wield with the flint locks.

Each Cartridge deals 30 piercing damage.
## Technical details

YAML Only
## Media

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl: Tryded
- add: Added a way for muskets and musket cartridges (NOT .60) to be crafted.

### CODE REVIEW

**Comment by Tryded** [comment]
Still trying to figure out how to make it reload slower without changing the firerate as the firerate makes it so that it is basically impossible to quickdraw it even when it is already reloaded.

**Comment by Tryded** [comment]
Ready for review

**Comment by Bonktrauma** [comment]
> Ready for review

Will it still allow .60?

**Comment by Tryded** [comment]
> Will it still allow .60?

It uses it's own musket cartridge which deals 30 pierce and is craftable

**Comment by Tryded** [comment]
hmm

**Comment by Tryded** [comment]
Hm.. It's been an while

**Comment by Tryded** [comment]
Does anyone have concerns?

File: Resources/Textures/DeltaV/Objects/Misc/bayonet.rsi/meta.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Made by Pato Grone for SS14, editted by Tryded",
```
Comment (Colin-Tel) [inline Resources/Textures/DeltaV/Objects/Misc/bayonet.rsi/meta.json]:
```suggestion
  "copyright": "Made by Pato Grone for SS14, edited by Tryded",
```
:)

**Review by Colin-Tel — APPROVED** [review]

**Pull request merged**